src/pkg/cli/login.go

```diff
@@ -78,7 +78,7 @@ func InteractiveLogin(ctx context.Context, client client.FabricClient, gitHubCli
 	term.Info("Successfully logged in to", host, "("+tenant.String()+" tenant)")
 
 	if err := saveAccessToken(fabric, at); err != nil {
-		term.Warn("Failed to save access token:", err)
+		term.Warnf("Failed to save access token, try re-logging in: %v", err)
```

Comment:
"re-logging in" feels a little awkward. How about "re-authenticating" or "logging in again"?

```suggestion
		term.Warnf("Failed to save access token, try re-authenticating: %v", err)
```